modules/tenant-management/src/Volo.Abp.TenantManagement.HttpApi/Volo/Abp/TenantManagement/TenantController.cs

```diff
@@ -18,41 +19,55 @@ public TenantController(ITenantAppService service)
             _service = service;
         }
 
+        [HttpGet]
+        [Route("{id}")]
         public virtual Task<TenantDto> GetAsync(Guid id)
         {
             return _service.GetAsync(id);
         }
 
+        [HttpGet]
         public virtual Task<PagedResultDto<TenantDto>> GetListAsync(GetTenantsInput input)
         {
             return _service.GetListAsync(input);
         }
 
+        [HttpPost]
         public virtual Task<TenantDto> CreateAsync(TenantCreateDto input)
         {
             return _service.CreateAsync(input);
         }
 
+        [HttpPut]
+        [Route("{id}")]
         public virtual Task<TenantDto> UpdateAsync(Guid id, TenantUpdateDto input)
         {
             return _service.UpdateAsync(id, input);
         }
 
+        [HttpDelete]
+        [Route("{id}")]
         public virtual Task DeleteAsync(Guid id)
         {
             return _service.DeleteAsync(id);
         }
 
+        [HttpGet]
+        [Route("{id}/DefaultConnectionString")]
         public Task<string> GetDefaultConnectionStringAsync(Guid id)
         {
             return _service.GetDefaultConnectionStringAsync(id);
         }
 
+        [HttpPut]
+        [Route("{id}/DefaultConnectionString")]
```

Comment:
`{id}/default-connection-string` would be better. same for GetDefaultConnectionStringAsync and DeleteDefaultConnectionStringAsync methods.
Want to use kebap-case wherever possible.